### ISSUE TITLE
Migrating from Travis

### ISSUE THREAD

**Issue by svgeesus** [WebAudio/web-audio-api#2357]
Previously, Travis CI was free. Now it is not, so W3C is migrating all W3C repos away from Travis to GitHub actions. Travis will stop working once our current credits are exhausted, which will apparently happen in a week or so at current usage rates (depending on how fast most repos migrate)

[Migrating from Travis CI to GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions)

**Comment by mdjp** [comment]
Unless there is an update on this we will start to migrate to github actions ourselves and ask for support if required.

**Comment by hoch** [comment]
There's a little complication for our case. The [simple workflow](https://github.com/w3c/spec-prod/blob/main/docs/examples.md#deploy-to-github-pages) can be applied to a well-formed bikeshed document, but we have several errors that are ignored by `expected-errs.txt` file.

Either 1) we can fix these errors and then use the recommended spec-prod workflow, or 2) devise a new GitHub action to continue to ignore these errors. I definitely prefer 1) since fixing these bikeshed errors won't show up as actual spec changes and we can benefit from the future spec-prod improvement.

**Comment by hoch** [comment]
Ideally, we want to follow what WebCodecs folks do:
https://github.com/w3c/webcodecs/blob/main/.github/workflows/auto-publish.yml

It clean and modular.

**Comment by hoch** [comment]
Bikeshed errors from the current spec:

```
WARNING: Couldn't determine width and height of this image: images/cancel-linear.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/cancel-linear.svg'
WARNING: Couldn't determine width and height of this image: images/cancel-setTarget.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/cancel-setTarget.svg'
WARNING: Couldn't determine width and height of this image: images/cancel-setValueCurve.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/cancel-setValueCurve.svg'
WARNING: Couldn't determine width and height of this image: images/channel-merger.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/channel-merger.svg'
WARNING: Couldn't determine width and height of this image: images/compression-curve.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/compression-curve.svg'
WARNING: Couldn't determine width and height of this image: images/dynamicscompressor-internal-graph.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/dynamicscompressor-internal-graph.svg'
WARNING: Couldn't determine width and height of this image: images/panner-coord.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/panner-coord.svg'
WARNING: Couldn't determine width and height of this image: images/cone-diagram.svg
cannot identify image file '/Users/hongchan/a/web-audio-api/images/cone-diagram.svg'
LINE ~1319: Couldn't find 'WEBCODECS' in bibliography data. Did you mean:
  webcontent
  websockets
  webworkers
  webmidi
  webrtc-dscp
LINE 2168: Can't find the 'contextOptions' argument of method 'OfflineAudioContext/constructor(numberOfChannels, length, sampleRate)' in the argumentdef block.
LINE 3303: Can't find the 'destinationNode' argument of method 'AudioNode/connect(destinationParam, output)' in the argumentdef block.
LINE 3303: Can't find the 'input' argument of method 'AudioNode/connect(destinationParam, output)' in the argumentdef block.
LINE 3404: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
LINE 3417: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
LINE 3432: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
LINE 3432: Can't find the 'input' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
LINE ~1256: No 'dfn' refs found for 'detached' with for='['BufferSource']'.
LINE ~2658: No 'dfn' refs found for 'write'.
LINE ~2707: No 'dfn' refs found for 'detached' with for='['BufferSource']'.
LINE ~4547: No 'dfn' refs found for 'write'.
LINE ~4547: No 'dfn' refs found for 'byte length' that are marked for export.
LINE ~4605: No 'dfn' refs found for 'write'.
LINE ~4605: No 'dfn' refs found for 'byte length' that are marked for export.
LINE ~4650: No 'dfn' refs found for 'write'.
LINE ~4688: No 'dfn' refs found for 'write'.
```

Some of errors (`argumentdef` block) are from a Bikeshed issue:
https://github.com/tabatkins/bikeshed/issues/1740

**Comment by hoch** [comment]
Here's the first attempt:
https://github.com/WebAudio/web-audio-api/pull/2377

**Comment by hoch** [comment]
Fixed by #2377.
